src/Console/Update.php

```diff
@@ -65,8 +65,14 @@ public function handle()
         $defaultCurrency = $this->currency->config('default');
 
         if ($this->input->getOption('exchangeratesapi')) {
+            if (!$api = $this->currency->config('api_key')) {
```

Comment:
This is a little messy. Please clean it up some to look more like this:

```
if ($access_key = $this->currency->config('api_key')) {
    return $this->updateFromExchangeRatesApi($defaultCurrency, $access_key);
}

$this->error('An API key is needed from exchangeratesapi.io to continue.');

return;
```

Also please make the variable more description, `$api` doesn't make any sense for the `access_key`, even `api_key` would be better.